docker-compose-generator/docker-fragments/opt-monero-expose.yml

```diff
@@ -0,0 +1,6 @@
+version: "3"
+
+services:
+  monerod:
+    ports:
+    - "127.0.0.1:18081:18081"
```

Comment:
can you add

```yaml
required:
  - "monero"
```

So people who use this fragment without using monero get a proper error message?